LICENSE

```diff
@@ -2186,3 +2186,28 @@ The externally maintained libraries used by Node.js are:
     NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
     SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
   """
+
+- synchronous-worker, located at lib/internal/vm/localworker.js, is licensed as follows:
```

Comment:
This needs to be updated to point to the correct location via the license builder.

Reply:
Could you point me at the docs for this?